packages/palette/src/elements/Text/Text.story.tsx

```diff
@@ -0,0 +1,182 @@
+import { storiesOf } from "@storybook/react"
+import { themeGet } from "@styled-system/theme-get"
+import React from "react"
+import styled from "styled-components"
+import { Color } from "../../Theme"
+import { Flex } from "../Flex"
+import { Text, TEXT_TREATMENTS, TEXT_VARIANTS } from "./Text"
+
+const Table = styled.table`
+  width: 100%;
+  border: 1px solid ${themeGet("colors.black10")};
+  border-collapse: collapse;
+
+  > thead > tr > th {
+    text-align: left;
+    font-weight: normal;
+  }
+
+  > thead > tr > th,
+  > tbody > tr > td {
+    border-bottom: 1px solid ${themeGet("colors.black10")};
+    border-left: 1px solid ${themeGet("colors.black10")};
+    padding: ${themeGet("space.1")};
```

Comment:
We should just be using `themeGet` instead of our custom helpers. For instance here the spacing value gets converted into px automatically, and in the case of colors, if we ever want to support a dark mode, we'll have to convert to this usage.

Reply:
@zephraph - I'm trying to remember the thinking behind our helpers -- I think it had to do with types, as `themeGet` isnt' type-safe.

Reply:
as in you could pass it any string? We could also just also just wrap it with types for each respective theme key. The critical aspect here is that we get the value through the theme in the props, which could change (as in the case of dark mode) rather than the static theme.

Reply:
A lot of the faffing about we've done with helpers were for types. So long as we can preserve type safety I'm not opposed to using themeGet.

Reply:
That said, I _do_ like the explicitness of the helpers and I don't think using them as any significant disadvantage over `themeGet` (given that their implementation could just as simply be altered to use `themeGet` under the hood).

Reply:
Yeah, I think that makes sense, I guess ultimately I wonder how often if at all we use these helpers outside of a styled-components context.